content/docs/2.5/scalers/gcp-pub-sub.md

```diff
@@ -29,14 +29,15 @@ The `credentialsFromEnv` property maps to the name of an environment variable in
 Here's an [example](https://github.com/kedacore/sample-go-gcppubsub).
 
 ### Authentication Parameters
-You can use `TriggerAuthentication` CRD to configure the authenticate by providing the service account credentials in JSON. 
+There are three options to configure authentication:
```

Comment:
I would prefer to keep this flavor of documentation to stay aligned with other scalers

```suggestion
There are three options to configure authentication:

**Credential based authentication:**

- `GoogleApplicationCredentials` - Service account credentials in JSON.

**Pod identity based authentication:**
- Use pod identity (`podIdentity`) to use the Service Account from the GCP metadata, of either the machine or [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
```